### ISSUE TITLE
2.0.0 renamed module

### ISSUE THREAD

**Issue by robnagler** [michaelherold/pyIsEmail#30]
```sh
$ pip install pyIsEmail==2.0.0
$ python -c 'import pyisemail'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'pyisemail'
$ python -c 'import pyIsEmail'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/vagrant/.pyenv/versions/py3/lib/python3.7/site-packages/pyIsEmail/__init__.py", line 1, in <module>
    from pyisemail.__about__ import __version__
ModuleNotFoundError: No module named 'pyisemail'
```

**Comment by michaelherold** [comment]
I'm quite confused as to how that happened. Something must have changed when I packaged it. Thank you for the report.

**Comment by abtechbit** [comment]
Same story here, getting import errors.

When using poetry it install into ".venv/lib/python3.8/site-packages/pyIsEmail" (case sensitive). If I manually rename the folder to "pyisemail" (all lowercase) it starts working again.

**Comment by michaelherold** [comment]
I have been working on this between illness and having company. I finally have a solution in #32 and will be releasing a new version shortly. It would be nice if I could push a fixed v2.0.0, but I don't think PyPI allows that (nor do I think they should). As such, it will likely be v2.0.1 that fixes the issue.

I'll update you when I have pushed. Thanks for your patience.

**Comment by michaelherold** [comment]
As suspected, I wasn't able to update v2.0.0. You should be able to use v2.0.1 now, however. Thank you both for the reports!
